src/App/Module.php

```diff
@@ -108,21 +104,13 @@ public function getName()
 	}
 
 	/**
-	 * @return string The base class name
+	 * @return ICanHandleRequests The base class name
```

Comment:
```suggestion
	 * @return ICanHandleRequests The module object
```
Should it still be called `getClass` then?